exporter/googlecloudpubsubexporter/config.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlecloudpubsubexporter
+
+import (
+	"fmt"
+	"regexp"
+
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+var topicMatcher = regexp.MustCompile(`projects/[a-z][a-z0-9\-]*/topics/.*`)
+
+type Config struct {
+	config.ExporterSettings `mapstructure:",squash"`
+	// Timeout for all API calls. If not set, defaults to 12 seconds.
+	exporterhelper.TimeoutSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+	exporterhelper.QueueSettings   `mapstructure:"sending_queue"`
+	exporterhelper.RetrySettings   `mapstructure:"retry_on_failure"`
+	// Google Cloud Project ID where the Pubsub client will connect to
+	ProjectID string `mapstructure:"project"`
```

Comment:
Can this be made optional in the same way as the GCP exporter (i.e. default to the project under which the code is running)? If this is already the case, please document that in the README and in the comment here.

Reply:
Changed in README